README.md

```diff
@@ -397,12 +397,7 @@ CPMAddPackage("gh:jbeder/yaml-cpp#yaml-cpp-0.6.3@0.6.3")
 ### [nlohmann/json](https://github.com/nlohmann/json)
 
 ```cmake
-CPMAddPackage(
-  NAME nlohmann_json
-  VERSION 3.9.1
-  GITHUB_REPOSITORY nlohmann/json
-  OPTIONS
-    "JSON_BuildTests OFF"
+CPMAddPackage("gh:nlohmann/json@3.9.1" OPTIONS "JSON_BuildTests OFF")
 )
```

Comment:
I actually find this slightly harder to read than before, maybe it would be better to keep some of the indentation?

```suggestion
CPMAddPackage("gh:nlohmann/json@3.9.1"
  OPTIONS "JSON_BuildTests OFF"
)
```